src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM_SAIL = 20;
```

Comment:
Potentially incorrect variable naming. Variable 'SHORT_TERM_SAIL' should be 'SHORT_TERM_SALE', since 'sale' is connected with discounts, while 'sail' is for boats.